src/main/java/com/connexta/search/index/IndexManagerImpl.java

```diff
@@ -67,7 +67,7 @@ public void index(
 
     log.info("Attempting to index product id {}", productId);
     try {
-      crudRepository.save(new Index(productId, contents));
+      crudRepository.save(new Index(productId, contents, mediaType));
```

Comment:
@connexta/i2o-cdr Passing every attribute into Index constructor won't look good after a bunch of terms are added. Add some point another strategy is needed, like setters or a builder.